src/locales/ko/mystery-encounters/bug-type-superfan-dialogue.json

```
@@ -1,38 +1,38 @@
 {
-  "intro": "An unusual trainer with all kinds of Bug paraphernalia blocks your way!",
-  "intro_dialogue": "Hey, trainer! I'm on a mission to find the rarest Bug Pokémon in existence!$You must love Bug Pokémon too, right?\nEveryone loves Bug Pokémon!",
-  "title": "The Bug-Type Superfan",
-  "speaker": "Bug-Type Superfan",
-  "description": "The trainer prattles, not even waiting for a response...\n\nIt seems the only way to get out of this situation is by catching the trainer's attention!",
-  "query": "What will you do?",
+  "intro": "온갖 종류의 곤충채집 장비를 가진\n특이한 트레이너가 길을 막아섰다!",
+  "intro_dialogue": "거기, 트레이너!\n난 세상에서 가장 희귀한 벌레 포켓몬을 찾고있어!$너도 벌레 타입 포켓몬 좋아하지?\n모두 벌레 타입 포켓몬을 좋아하니깐!",
+  "title": "벌레타입 포켓몬 매니아",
+  "speaker": "벌레타입 매니아",
+  "description": "이 트레이너는 대답할 틈도 없이, 계속 떠들어대고 있습니다…\n\n이 상황에서 벗어나려면, 트레이너의 주의를 돌려야 할 것 같습니다!",
+  "query": "어떻게 하시겠습니까?",
   "option": {
     "1": {
-      "label": "Offer to Battle",
-      "tooltip": "(-) Challenging Battle\n(+) Teach a Pokémon a Bug Type Move",
-      "selected": "A challenge, eh?\nMy bugs are more than ready for you!"
+      "label": "승부한다",
+      "tooltip": "(-) 포켓몬 배틀 도전\n(+) 벌레타입 기술을 가르침",
+      "selected": "나한테 도전하겠다고?\n내 벌레 포켓몬들은 이미 준비됐어!"
     },
     "2": {
-      "label": "Show Your Bug Types",
-      "tooltip": "(+) Receive a Gift Item",
-      "disabled_tooltip": "You need at least 1 Bug Type Pokémon on your team to select this.",
-      "selected": "You show the trainer all your Bug Type Pokémon...",
-      "selected_0_to_1": "Huh? You only have {{numBugTypes}}...$Guess I'm wasting my breath on someone like you...",
-      "selected_2_to_3": "Hey, you've got {{numBugTypes}} Bug Types!\nNot bad.$Here, this might help you on your journey to catch more!",
-      "selected_4_to_5": "What? You have {{numBugTypes}} Bug Types?\nNice!$You're not quite at my level, but I can see shades of myself in you!\n$Take this, my young apprentice!",
-      "selected_6": "Whoa! {{numBugTypes}} Bug Types!\n$You must love Bug Types almost as much as I do!$Here, take this as a token of our camaraderie!"
+      "label": "벌레 포켓몬을 보여준다",
+      "tooltip": "(+) 아이템 획득",
+      "disabled_tooltip": "벌레타입 포켓몬이 없다.",
+      "selected": "트레이너에게 가지고 있는 모든 벌레 포켓몬을 보여주었다...",
+      "selected_0_to_1": "응? 너 {{numBugTypes}}마리 갖고 있구나...$너 같은 일반인한테 떠들다었다니, 산소가 아깝네…",
```

```suggestion
      "selected_0_to_1": "응? 너 {{numBugTypes}}마리밖에 안 갖고 있구나...$너 같은 일반인한테 떠들었다니, 쓸데없이 입만 아팠네…",
```
'산소가 아깝다'라는 표현이 한국어로 조금 직관적이지 않은 듯해서 사전에서 'waste one's breath'의 해석으로 제시한 '입만 아프다'라는 표현을 제안해 봅니다.